src/utils/authentication.ts

```diff
@@ -19,14 +25,16 @@ async function maybeSaveFakeAuthKey() {
 }
 
 export async function isAuthenticated() {
-  if (IS_IOS) {
+  const hasBiometricsEnabled = await getSupportedBiometryType();
+  if (hasBiometricsEnabled) {
```

Comment:
Might this be true for Android also?

Reply:
yes it should be 

only want to fallback to pin on android